samples/04_gis_analysts_data_scientists/tabular_data_supervised_learning_using_automl.ipynb

```diff
@@ -0,0 +1,1201 @@
+{
```

Comment:
<p>Better to change login to <code>gis=GIS('home')</code>.  Then ensure the data layer is shared publicly</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/977/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Done</p>